responses/04_good-template.md

```diff
@@ -0,0 +1 @@
+Nice template. I'm going to merge this for you. You can find your next steps in [your next pull request]({{ repoUrl }}/pull/3).
```

Comment:
I had a conflict.  The bot responded with "I'm going to merge this for you" but it can't because of the conflict. Can we make this step smarter by checking if the branch is mergeable?

Reply:
Tried this here: f9a2715